src/decode.rs

```diff
@@ -1453,13 +1453,11 @@ unsafe fn decode_b(
 
     let ts = &mut *t.ts;
     let f = &mut *t.f;
+    let decode_block_info = DEBUG_BLOCK_INFO(f, t);
```

Comment:
Can you name this just `dbg`?  I named a similarly hoisted variable the same before.

Reply:
Also, to answer your question, it should be safe to precompute like this if `f.bx` and `f.by` don't change at all in the function.

Reply:
`dbg` works for me 👍

And I did check `decode_b` to see if `f.bx`/`by` change, and it doesn't seem like it? I see it being modified in `decode_sb` and a few other places, but not `decode_b`.